setup.py

```diff
@@ -1,5 +1,12 @@
+# Checking python version: If python 3 was not found then it returns 1 and does
+#   not do anything
+from platform import python_version
+primer_version = python_version().split(".")
+if int(primer_version[0]) != 3:
+    print ("This module only works with python3")
+    print ("To setup the module simply run `python3 setup.py install`")
+    exit(1)
 
```

Comment:
same as what you have done on master. I was not aware of that. You have access to merge, so you can decide what to use instead of what.